platform/pom.xml

```diff
@@ -367,6 +367,7 @@
         <module>landing-page</module>
         <module>platform-app</module>
         <module>docs</module>
+        <module>platform-api</module>
```

Comment:
This probably shouldn't be the last thing to be built in platform.  It also looks like you are never deploying this bundle via the platform-app features.xml file.